src/components/Input/RadioButton/RadioGroup.js

```diff
@@ -3,22 +3,38 @@ import PropTypes from "prop-types";
 import Provider from "../../SelectionProvider/Provider";
 import KeyBoardProvider from "../../KeyboardNavigation/Provider";
 
-const RadioGroup = ({ onChange, children, value }) => (
+const RadioGroup = ({
+  onChange,
+  children,
+  value,
+  ariaLabel,
+  ariaLabelledBy
```

Comment:
Maybe pass any additional props passed to `RadioGroup` to `KeyBoardProvider`, rather than adding `ariaLabel` and `ariaLabelledBy` individually?